ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1576,6 +1576,11 @@ vxlan_interface:
       mlag_source_interface: < source_interface_name >
       udp_port: < udp_port >
       virtual_router_encapsulation_mac_address: < mlag-system-id | ethernet_address (H.H.H) >
+      bfd_vtep_evpn:
+        interval: < integer >
+        min_rx: < integer >
+        multiplier: < 3-50 >
+        prefix_list: < prefix-list >
```

Comment:
I know we are not consistent everywhere in the README but it seems we are in favor of `_` instead of `-`

```suggestion
        prefix_list: < prefix_list >
```